tests/integration/test_charm.py

```diff
@@ -53,12 +53,30 @@ async def test_discourse_up(ops_test: OpsTest, app: Application, requests_timeou
     session.mount("http://", HTTPAdapter(max_retries=retries))
     response = session.get(
         f"http://{address}:{SERVICE_PORT}/finish-installation/register",
-        headers={"Host": f"{app.name}.local"},
         timeout=requests_timeout,
     )
     assert response.status_code == 200
 
 
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_prom_exporter_is_up(app: Application):
+    """
+    arrange: given charm in its initial state
+    act: when the metrics endpoint is scraped
+    assert: the response is 200 (HTTP OK)
+    """
+    # Application actually does have units
+    indico_unit = app.units[0]  # type: ignore
+    cmd = "curl http://localhost:9394/metrics"
```

Comment:
Perhaps use a constant here instead of hardcoding the port on both sides ?

Reply:
Done